pf/internal/convert/convert.go

```diff
@@ -90,19 +91,19 @@ func NewConfigPropertyNames(pn PropertyNames) LocalPropertyNames {
 }
 
 type Encoder interface {
-	FromPropertyValue(resource.PropertyValue) (tftypes.Value, error)
+	fromPropertyValue(resource.PropertyValue) (tftypes.Value, error)
```

Comment:
These are made internal since they're not secret-aware; to make sure the secret handling does not leak outside this module.